pyqt-apps/siriushla/widgets/matplotlib.py

```diff
@@ -47,11 +47,20 @@ def changeEvent(self, event):
 class MatplotlibWidget(QWidget):
     """MatplotlibWidget class."""
 
-    def __init__(self, figure=None, parent=None):
+    def __init__(self, figure=None, parent=None, toolbar_position='bottom'):
         """."""
         super().__init__(parent)
         self.setLayout(QVBoxLayout())
-        self.canvas = MatplotlibCanvas(figure, self)
+        self.figure = figure or Figure()
+        self.canvas = MatplotlibCanvas(self.figure, self)
+        if toolbar_position is None:
+            self.navtool = None
+            return
```

Comment:
before return, you need to add `self.canvas` to layout

Reply:
right!

Reply:
done!